sdk/tables/Azure.Data.Tables/src/Extensions/TableEntityExtensions.cs

```diff
@@ -60,6 +60,10 @@ internal static class TableEntityExtensions
             // Remove the ETag property, as it does not need to be serialized
             annotatedDictionary.Remove(TableConstants.PropertyNames.ETag);
 
+            // Remove the Timestamp property, as it is controlled by the service not the client.
```

Comment:
Let's also remove the `TimeStamp` in the case where `entity` is an `IDictionary` [here](https://github.com/Azure/azure-sdk-for-net/blob/d0df5854409be309dd907a9cbbbce252824767fa/sdk/tables/Azure.Data.Tables/src/Extensions/DictionaryTableExtensions.cs#L33)

Reply:
Done.